pages/patterns/delete-pattern.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { Heading } from '@twilio-paste/core/heading';
+import { Text } from '@twilio-paste/core/text';
+import { Button } from '@twilio-paste/core/button';
+import { Flex } from '@twilio-paste/core/flex';
+import { Paragraph } from "@twilio-paste/core/paragraph";
+import { Table, TBody, Th, THead, Tr, Td } from '@twilio-paste/core/table';
+import { useToaster, Toaster } from "@twilio-paste/core/toast";
+import { DeleteIcon } from '@twilio-paste/icons/cjs/DeleteIcon';
+
+export default function Home() {
+
+    const [data, setData] = React.useState([
+        {
+            id: 1,
+            friendlyName: 'first service',
+            sid: 4392908903
+        },
+        {
+            id: 2,
+            friendlyName: 'second service',
+            sid: 8439025738
+        },
+        {
+            id: 3,
+            friendlyName: 'third service',
+            sid: 8439025703
+        },
+        {
+            id: 4,
+            friendlyName: 'fourth service',
+            sid: 6239485739
+        }
+    ])
+
+    const toaster = useToaster();
+
+    const handleDelete = (service) => {
+        toaster.push({
+            message: 'Service was successfully deleted.',
+            variant: 'success',
+            dismissAfter: 3000
+        })
+
+        console.log('service:', service.id)
+        const updatedServices = data.filter(serviceObj => serviceObj.id != service.id)
+        setData(updatedServices)
+    }
```

Comment:
Really cool to see this working!
I'm not sure if my comment  is within the scope of this PR but when I click on delete, the notification has the box around the X selected already. 
![Screen Shot 2021-01-14 at 11 40 09 AM](https://user-images.githubusercontent.com/75345983/104641725-ff980700-565e-11eb-87f3-0d193e1ee556.png)

Reply:
I tried changing this by adding `setFocus: false` on the toaster, but it didn't make a difference. I wonder if it's automatically set to focus on the X for accessibility reasons? @SiTaggart is there a reason for this / does it matter?